Src/BootCamp.Chapter/PeoplePredicates.cs

```diff
@@ -10,20 +10,21 @@ public static class PeoplePredicates
         /// a) over 18, who do not live in UK, whose surename does not contain letter 'a'.
         /// </summary>
         /// <returns></returns>
-        public static bool IsA(Person person) => false;
+        public static bool IsA(Person person) => person.CalculateAge() > 18 &  person.Country != "UK" & !person.SureName.Contains('a')  ;
```

Comment:
So the question is a bit difficult but.. there are no "UK"s in the data file. It's actually `United Kingdom`. So you've never actually filtered out people from the UK.

This is where a better data type would come in handy :D. 

also, if there was `Uk` this would still fail due to case sensitivity. So you would want to use `.Equals("", StringComparison.OrdinalIgnoreCase)`

Reply:
I have just reviewed the tests actually and Kai used UK in the test data so this is okay. I would still use `.Equals` for safty against case sensitivity :)

Reply:
changed